mitodl/mitxpro
# SEO metadata for product pages

#### Pre-Flight checklist


#### What are the relevant tickets?
FIxes #270 

#### What's this PR do?
Add wagtail plugin https://github.com/neon-jungle/wagtail-metadata for SEO metadata.

#### How should this be manually tested?
Render product page, you should see following tags in `<head>`
```
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="seo title of the page — None">
<meta name="twitter:description" content="seo description of the page.">
<meta property="og:url" content="http://edx.odl.local:8053/seed-models-in-engineering/">
<meta property="og:title" content="seo title of the page">
<meta property="og:description" content="seo description of the page.">
<meta property="og:site_name" content="None">
<meta itemprop="url" content="http://edx.odl.local:8053/seed-models-in-engineering/">
<meta itemprop="name" content="seo title of the page — None">
<meta itemprop="description" content="seo description of the page.">
<meta name="description" content="seo description of the page.">
```
You can also edit some tags through `promote_panels` in wagtail.

#### Any background context you want to provide?
~~We have decided to add these tags only to product pages (course and program) and Home page #323 . Home page is still in development phase, I have asked @ahmed-arbisoft to add `meta_tags` in his PR.
It that's not possible, I will create new PR or update this PR after #323 merged.~~

**Note:**
PR updated for adding metadata in homepage as well.
